yt/chyt/server/config.cpp

```diff
@@ -140,6 +140,10 @@ void TExecutionSettings::Register(TRegistrar registrar)
 
     registrar.Parameter("enable_min_max_filtering", &TThis::EnableMinMaxFiltering)
         .Default(true);
+
+    // TODO(achulkov2): Set to false by default. This is temporary to run all tests and verify that nothing fails.
+    registrar.Parameter("enable_optimize_read_in_order", &TThis::EnableOptimizeReadInOrder)
+        .Default(true);
```

Comment:
This is set to `true` temporarily, so that we run all CHYT tests in the import and see that nothing breaks.

Reply:
> Why not to use native ClickHouse's enable_optimize_read_in_order option?
Do we have the ability to enable/disable native CH-options for the whole clique?

Reply:
>Did you measure the worst possible performance degradation from this optimization? If the degradation is not significant, we can leave it on by default.

What is the best way to go about measuring this? Do we have perf tests? :)

Reply:
> Do we have the ability to enable/disable native CH-options for the whole clique?

Yes, we do. The default CH settings may be specified in the cliques's config like this: `{clickhouse={settings={enable_optimize_read_in_order=1}}}`

But afaik, it's turned on by default in CH

> What is the best way to go about measuring this? Do we have perf tests? :)

Unfortunately, we don't have perf tests yet. I think that a simple query on a large table, with all the rows being filtered out, would give us an idea of how bad the performance degradation might be.

Reply:
Since we want to disable the option by default for now, I will keep our separate option. I don't think it is easy (or that we should) disable a native CH option from our code. 

If we decide to enable it by default, we should probably deprecate it in favor of the native option (which is turned on by default).

Reply:
Ok